ui/css/module/_styleguide.scss

```diff
@@ -0,0 +1,26 @@
+.styleguide-container {
+  padding: 1rem;
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.styleguide-example {
+  border: 1px solid $color-lt-gray;
+  padding: 1rem 2.5rem;
+
+  h4 {
+    text-transform: uppercase;
+    color: $color-gray;
+    letter-spacing: 1px;
+    font-size: 1em;
```

Comment:
Will this override the nested h4 in `styleguideTypographyHeading`? If so, can we just remove these rules? They don't seem super important.